app/models/assignment.rb

```diff
@@ -21,6 +21,7 @@ class Assignment < ActiveRecord::Base
 
   validates :title, presence: true
   validates :title, uniqueness: { scope: :organization }
+  validates :title, length: { maximum: 100 }
```

Comment:
Let's make this 60 for now. Maximum username length on GitHub is 39, plus a character for the separator. We can always increase the limit later if we want.